src/distilabel/distiset.py

```diff
@@ -225,6 +224,7 @@ def create_distiset(  # noqa: C901
         correspond to different configurations of the dataset.
     """
     logger = logging.getLogger("distilabel.distiset")
+    from distilabel.steps.constants import DISTILABEL_METADATA_KEY
```

Comment:
Could we make this the first line within the method?

```suggestion
    from distilabel.steps.constants import DISTILABEL_METADATA_KEY
    
    logger = logging.getLogger("distilabel.distiset")
```